aws-amp-integration.md

```diff
@@ -9,136 +9,271 @@ Kubecost leverages the open-source Prometheus project as a time series database
+Your Amazon EKS clusters have [Amazon EBS CSI](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) driver installed
+### Create Amazon Managed Service for Prometheus workspace:
+
+#### Step 1: run the following command to get the information of your current EKS cluster:
+
+```bash
+kubectl config current-context
 ```
+The example output should be in this format:
 
-### Setting up AMP
+```bash
+arn:aws:eks:${AWS_REGION}:${YOUR_AWS_ACCOUNT_ID}:cluster/${YOUR_CLUSTER_NAME}
+```
 
-You can use the following AWS CLI command to create a new AMP workspace:
+#### Step 2: run the following command to create new a Amazon Managed Service for Prometheus workspace
 
 ```bash
-export AWS_REGION=<YOUR-AWS-REGION>
+export AWS_REGION=<YOUR_AWS_REGION>
 aws amp create-workspace --alias kubecost-amp --region $AWS_REGION
 ```
 
-Example output:
+The Amazon Managed Service for Prometheus workspace should be created in a few seconds. Run the following command to get the workspace ID:
 
-```json
-{
-    "arn": "arn:aws:aps:us-west-2:xxxxxxxxxxxxx:workspace/${AMP_WORKSPACE_ID}",
-    "status": {
-        "statusCode": "CREATING"
-    },
-    "tags": {},
-    "workspaceId": "${AMP_WORKSPACE_ID}"
-}
+```bash
+export AMP_WORKSPACE_ID=$(aws amp list-workspaces --region ${AWS_REGION} --output json --query 'workspaces[?alias==`kubecost-amp`].workspaceId | [0]' | cut -d'"' -f 2)
+echo $AMP_WORKSPACE_ID
 ```
+### Set up the environment:
+#### Step 1: set environment variables for integrating Kubecost with Amazon Managed Service for Prometheus
 
-The workspace should be created in a few seconds. You can log in to the [AWS AMP console](https://console.aws.amazon.com/prometheus/) to retrieve more information. You need to set `$REMOTEWRITEURL` and `$QUERYURL` for use in the integration with Kubecost later as follows:
+Run the following command to set environment variables for integrating Kubecost with Amazon Managed Service for Prometheus
 
-* `REMOTEWRITEURL="https://aps-workspaces.us-west-2.amazonaws.com/workspaces/${AMP_WORKSPACE_ID}/api/v1/remote_write"`
-* `QUERYURL="http://localhost:8005/workspaces/${AMP_WORKSPACE_ID}"`
+```bash
+export RELEASE="kubecost"
+export YOUR_CLUSTER_NAME=<YOUR_EKS_CLUSTER_NAME>
+export AWS_REGION=${AWS_REGION}
+export VERSION="1.103.4"
+export KC_BUCKET="kubecost-etl-metrics" # Remove this line if you want to set up small-scale infrastructure 
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export REMOTEWRITEURL="https://aps-workspaces.${AWS_REGION}.amazonaws.com/workspaces/${AMP_WORKSPACE_ID}/api/v1/remote_write"
+export QUERYURL="http://localhost:8005/workspaces/${AMP_WORKSPACE_ID}"
+```
+#### Step 2: set up S3 bucket, IAM policy and Kubernetes secret for storing Kubecost ETL files
 
-## Install Kubecost with the default values
+Note: You can ignore this step 2 for the small-scale infrastructure setup
```

Comment:
"> **Note**: You can ignore Step 2 when setting up for small-scale infrastructure."

Reply:
Thank you Brett ! Fixed in the latest commit.